prediction_market_agent/agents/prophet_agent/deploy.py

```diff
@@ -58,6 +69,44 @@ def load(self) -> None:
         )
 
 
+class DeployablePredictionProphetGPT4oAgentNewMarketTrader(
+    DeployablePredictionProphetGPT4oAgent
+):
+    """
+    This agent trades on new markets, then re-evaluates the market over its
+    lifetime, if it observes that news has been published about the market since
+    its last trade.
+    """
+
+    bet_on_n_markets_per_run = 20
+    trade_on_markets_created_after = DatetimeUTC(2024, 10, 31, 0)
+    get_markets_sort_by = SortBy.NEWEST
+    same_market_trade_interval = MarketLifetimeProportionalInterval(max_trades=4)
+    relevant_news_response_cache = RelevantNewsResponseCache(
+        sqlalchemy_db_url=APIKeys().sqlalchemy_db_url.get_secret_value()
```

Comment:
This is initialised at import-time, it will error out if sqlalchemy_db_url is not defined in the environment, I think that is the case for some agents. (`load` method can be used for this)